### PR TITLE
Version and docs

### PR DESCRIPTION
**Pull request by peter-kehl**

### CODE REVIEW

**Comment by peter-kehl** [comment]
Dear Matthias,

Thank you for Javacc-Gradle.
This is to make it easier for users.

File: README.asciidoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = JavaCC Gradle Plugin
-:latestRevision: 3.0.1
+:latestRevision: 3.0.3
```
Comment (martinda) [inline README.asciidoc]: You could make this 3.1.1.

**Comment by martinda** [comment]
Thanks @peter-kehl, this is a good PR, it took me a few hours to figure out the same thing. @m-raab please accept it, it would have saved me time. Thanks in advance.

**Comment by m-raab** [comment]
the documentation was updated for the latest version 3.1.1

**Pull request closed without merging**